cmd/run.go

```diff
@@ -23,29 +26,40 @@ var cmdRun = &cobra.Command{
 	Use:   "run",
 	Short: "Run flasher service to listen for events and install firmware",
 	Run: func(cmd *cobra.Command, _ []string) {
-		runWorker(cmd.Context())
+		var mode model.RunMode
```

Comment:
Did you a specific approach in mind here - since I do have the flags marked as mutually exclusive - https://github.com/metal-toolbox/flasher/pull/182/files/4a72d802c7d4c6603480f23a30eab371e82eea62#diff-8146f8148ccbf6711d65f532f6ab9a7c8dfbdc3960c7ffc974f17d6d224dd349R195-R196